runtime/integration-tests/src/cases.rs

```diff
@@ -8,10 +8,10 @@ mod investments;
 mod liquidity_pools;
 mod liquidity_pools_gateway_queue;
 mod loans;
-mod lp;
```

Comment:
@William TODO

Reply:
I will take care of this

Reply:
It was pretty straighforward: [6c1b30f](https://github.com/centrifuge/centrifuge-chain/pull/1958/commits/6c1b30f1fe51c4e824da16b0ad0a8cafa274ab12)

Reply:
Cant find the commit you have linked but I am happy to hear it wasn't a nightmare!

Reply:
I rebase changed the hash 😅